src/game/client/neo/ui/neo_hud_message.h

```diff
@@ -27,12 +29,15 @@ class CNEOHud_Message : public CNEOHud_ChildElement, public CHudElement, public
     virtual ConVar* GetUpdateFrequencyConVar() const override;
 
 private:
+    std::wstring ProcessKeyBinds(wchar_t* rawmessage);
+
```

Comment:
This can be const-qualified as
```cpp
std::wstring ProcessKeyBinds(const wchar_t* rawmessage) const;
```
But do consider using `UTIL_ReplaceKeyBindings` instead.